### ISSUE TITLE
Wrong profile picture

### ISSUE THREAD

**Issue by onuryuxcodes** [stagecast/bug-report#99]
**Describe the bug**
I get the wrong profile picture if I login with another account.

**To Reproduce**
1. Login with facebook and see facebook profile picture in the app
2. Log out and sign up with another user.
3. Kill the app and restart
4. See facebook profile picture of the first account in the app

**Expected behavior**
Don't see the previous profile picture

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Device: [iPhone 7]
 - OS: [iOS11.4.1]

**Additional context**
The account seems to be correct (at least from what i can see, the name, friends etc. are correct) just the profile pic is wrong after the steps

**Comment by ConradoMateu** [comment]
Cannot reproduce this, maybe with the fix of the StagecastDev and Stagecast Facebook Login issue this is working now.

**Comment by mawlle** [comment]
Cannot reproduce this error as I can't login with Facebook. 

This is my flow:
1. Press "Login with facebook" in the sign up journey
2. Get redirectet to a facebook web page - Press continue
3. Get back to the app but get an error message (see below)

![img_4120](https://user-images.githubusercontent.com/18073152/50889552-8ecb0e00-13f8-11e9-8295-cd3febdd5bf9.PNG)

**Comment by ConradoMateu** [comment]
Are you using StagecasDEV? if so there is a configuration needed cause before was messing with the configuration of Stagecast in production so I have to split it in 2 different environments. See https://github.com/stagecast/iOS/issues/464

**Comment by mawlle** [comment]
Yep, I'm using 2.1.8 StagecastDEV. Should we avoid testing the facebook login in the StagecastDEV version or what is the solution to the problem?
